selfhost/types.jakt

```diff
@@ -1062,34 +1090,66 @@ enum BlockControlFlow {
         PartialNeverReturns(might_break: this_might_break) => match second {
             NeverReturns | AlwaysReturns => BlockControlFlow::PartialNeverReturns(might_break: this_might_break)
             MayReturn => BlockControlFlow::PartialNeverReturns(might_break: this_might_break)
-            AlwaysTransfersControl(might_break) => BlockControlFlow::PartialAlwaysTransfersControl(might_break: might_break or this_might_break)
-            PartialNeverReturns(might_break) => BlockControlFlow::PartialNeverReturns(might_break: might_break or this_might_break)
-            PartialAlwaysReturns(might_break) => BlockControlFlow::PartialAlwaysReturns(might_break: might_break or this_might_break)
-            PartialAlwaysTransfersControl(might_break) => BlockControlFlow::PartialAlwaysTransfersControl(might_break: might_break or this_might_break)
+            AlwaysTransfersControl(
+                might_break
+            ) => BlockControlFlow::PartialAlwaysTransfersControl(might_break: might_break or this_might_break)
+            PartialNeverReturns(
+                might_break
+            ) => BlockControlFlow::PartialNeverReturns(might_break: might_break or this_might_break)
+            PartialAlwaysReturns(
+                might_break
+            ) => BlockControlFlow::PartialAlwaysReturns(might_break: might_break or this_might_break)
+            PartialAlwaysTransfersControl(
+                might_break
+            ) => BlockControlFlow::PartialAlwaysTransfersControl(might_break: might_break or this_might_break)
         }
         PartialAlwaysReturns(might_break: this_might_break) => match second {
             NeverReturns | AlwaysReturns => BlockControlFlow::PartialAlwaysReturns(might_break: this_might_break)
             MayReturn => BlockControlFlow::PartialAlwaysReturns(might_break: this_might_break)
-            AlwaysTransfersControl(might_break) => BlockControlFlow::PartialAlwaysTransfersControl(might_break: might_break or this_might_break)
-            PartialNeverReturns(might_break) => BlockControlFlow::PartialAlwaysReturns(might_break: might_break or this_might_break)
-            PartialAlwaysReturns(might_break) => BlockControlFlow::PartialAlwaysReturns(might_break: might_break or this_might_break)
-            PartialAlwaysTransfersControl(might_break) => BlockControlFlow::PartialAlwaysTransfersControl(might_break: might_break or this_might_break)
+            AlwaysTransfersControl(
+                might_break
```

Comment:
same comment as above. It seems it's not using the `=>` as the place to break at.